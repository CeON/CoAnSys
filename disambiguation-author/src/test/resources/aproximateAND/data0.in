c1	1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
