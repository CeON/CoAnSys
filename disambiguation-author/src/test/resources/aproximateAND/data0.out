({(c1,1,Kowalski,[EX_YEAR#{(2011)},EX_TITLE#{(An Empirical Study on the AND)}])},{})
